### PR TITLE
feat(core): apply default doc mode setting

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#8069** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8069?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8083** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8083?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
 import { EditorSetting } from '../entities/editor-setting';
 
+@OnEvent(WorkspaceInitialized, e => e.onWorkspaceInitialized)
 export class EditorSettingService extends Service {
   editorSetting = this.framework.createEntity(EditorSetting);
+
+  onWorkspaceInitialized(workspace: Workspace) {
+    // set default mode for new doc
+
+    workspace.docCollection.slots.docCreated.on(docId => {
```
Comment (forehalo) [inline packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts:18]: duplicated @JimmFly  ?

Comment (JimmFly) [reply packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts:18]: I can rebase to this branch

Comment (pengx17) [inline packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts:18]: is it possible in this time sequence?
1. doc created
2. doc is set to edgeless mode (using createEdgeless helper)
3. docCreated event emitted and then set to default mode stored in setting

The subtle part is that whether docCreated event is sent & captured in a single create doc call

Comment (forehalo) [reply packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts:18]: I prefer explicitly setting primary mode.

Comment (EYHN) [reply packages/frontend/core/src/modules/editor-settting/services/editor-setting.ts:18]: docCreated Should always be triggered first

**Review by pengx17 — APPROVED** [review]

**Pull request merged**